lib/agent/providers/geo/strategies.js

```diff
@@ -32,6 +33,12 @@ function geoip(cb) {
   });
 }
 
+var save_data = (data) => {
+  storage.do('set', {type: 'keys', id: 'geo', data: {value: JSON.stringify(data)}}, (err) => {
```

Comment:
we're only storing locations from the wifi strategy? then we should use a more specific name for that key, for example: `last_wifi_location`

Reply:
done